packages/holocron/src/holocronModule.jsx

```diff
@@ -134,8 +135,13 @@ export default function holocronModule({
         };
       }, []);
 
-      // eslint-disable-next-line react/jsx-props-no-spreading -- spread props
-      return <WrappedComponent {...props} moduleLoadStatus={status} />;
+      return (
+        // eslint-disable-next-line react/jsx-no-constructed-context-values -- name is static
```

Comment:
The rule isn't about the property changing, but the object identity changing at render time. 

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md

Examples of **incorrect** code for this rule:

```jsx
return (
    <SomeContext.Provider value={{foo: 'bar'}}>
        ...
    </SomeContext.Provider>
)
```

Examples of **correct** code for this rule:

```jsx
const foo = useMemo(() => ({foo: 'bar'}), []);
return (
    <SomeContext.Provider value={foo}>
        ...
    </SomeContext.Provider>
)
```

Reply:
Resolved in [feccb30](https://github.com/americanexpress/holocron/commit/feccb305b3e91ed1ef910423297a22790926b8a5)